docs/commands/viewport.md

```diff
@@ -9,54 +9,35 @@ To set the viewport size and dimension, use the `cy.viewport()` command.
 <!-- fiddle.export cy.viewport() - set the viewport size and dimension -->
 
 ```js
-cy.get('.navbar').should('be.visible')
-cy.viewport(320, 480)
+const viewportWidths = ['macbook-15', 'macbook-13', 'macbook-11', 'ipad-2', 'ipad-mini', 'iphone-6+', 'iphone-6', 'iphone-5', 'iphone-4', 'iphone-3'];
```

Comment:
I don't like the variable name "viewportWidths", since these are NOT widths, more like viewports or deviceViewports

Reply:
Actually, you're right, i updated it to be deviceViewports